### PR TITLE
fix: RTL localization for assets and strings

### PR DESCRIPTION
**Pull request by RawanMatar89**
### Describtion:
1. changed "direction" attribute for "arrowRight16" image in "Core/Assets" from attribute inspector.
2. localize "chevron.right" system image that used in: 
CourseButton
HandoutsView
CourseStructureView
CourseVerticalView
CourseDatesView
DiscussionTopicsView
ProfileSupportInfoView
SettingsView
using "flipsForRightToLeftLayoutDirection" method
3. localize "CompletionStatus" enum string by adding localized var that returns a localized string.

### Screenshots:
| Before | After |
|--------|--------|
|Simulator Screenshot - iPhone 15 Pro Max - 2024-05-23 at 09 50 23 | Simulator Screenshot - iPhone 15 Pro Max - 2024-05-23 at 09 55 57 |
|Simulator Screenshot - iPhone 15 Pro Max - 2024-05-23 at 09 50 43 |Simulator Screenshot - iPhone 15 Pro Max - 2024-05-23 at 09 56 10|
|Simulator Screenshot - iPhone 15 Pro Max - 2024-05-23 at 09 52 02| Simulator Screenshot - iPhone 15 Pro Max - 2024-05-23 at 09 57 15|
|Simulator Screenshot - iPhone 15 Pro Max - 2024-05-23 at 09 52 25|Simulator Screenshot - iPhone 15 Pro Max - 2024-05-23 at 09 56 18|

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @RawanMatar89! Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

**Comment by OmarIthawi** [comment]
@RawanMatar89 please check the test failures https://github.com/openedx/openedx-app-ios/actions/runs/9204972524/job/25319630671?pr=441

**Comment by OmarIthawi** [comment]
@RawanMatar89 please rebase over [openedx:develop](https://github.com/openedx/openedx-app-ios/tree/develop), resolve conflicts and fix failing tests mentioned in https://github.com/openedx/openedx-app-ios/pull/441#issuecomment-2139217324

File: Course/Course/Presentation/Dates/CourseDatesView.swift [hunk 1]
```diff
@@ -287,6 +287,8 @@ struct CompletedBlocks: View {
                                 if block.canShowLink && !block.firstComponentBlockID.isEmpty {
                                     Image(systemName: "chevron.right")
                                         .resizable()
+                                        .flipsForRightToLeftLayoutDirection(true)
+
```
Comment (OmarIthawi) [inline Course/Course/Presentation/Dates/CourseDatesView.swift:291]: Please remove the extra line here.

**Review by OmarIthawi — APPROVED** [review]
Thanks @RawanMatar89. The pull request looks good to me (as a non iOS engineer).

Please mark it as "Ready for review".

@volodymyr-chekyrta kindly triage this PR and let us know if you have any quesations.

**Review by volodymyr-chekyrta — APPROVED** [review]
LGTM

**Comment by openedx-webhooks** [comment]
@RawanMatar89 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/openedx-app-ios&entry.867055334=https://github.com/openedx/openedx-app-ios/pull/441&entry.1484655318=https://github.com/RawanMatar89&entry.752974735=2024-05-23+08:14&entry.1917517419=2024-07-25+08:24&entry.2133058324=Yes) so we can improve your experience in the future.

**Comment by volodymyr-chekyrta** [comment]
@RawanMatar89, thank you for your contribution!

**Pull request merged**